routing/index_graph_starter_joints.cpp

```diff
@@ -226,30 +230,31 @@ void IndexGraphStarterJoints::GetEdgeList(JointSegment const & vertex, bool isOu
 
   if (!isOutgoing)
   {
-    // |parentSegment| - это вершина-родитель от которой мы ищем детей
-    // Надо взять вес joint'a, который оканчивается в |parentSegment|
+    // |parentSegment| - it is parent-vertex, from which we searches children.
+    // For correct weight calculation we should get weight of JointSegment, that
+    // ends in |parentSegment| and add |parentWeight[i]| to the saved value.
     auto it = m_savedWeight.find(vertex);
     CHECK(it != m_savedWeight.end(), ("Can not find weight for:", vertex));
 
     Weight const & weight = it->second;
     for (size_t i = 0; i < edges.size(); ++i)
     {
-      // Сохраняем вес детей-joint'ов для последующих потомков.
-      //m_savedWeight[edges[i].GetTarget().GetSegment(false /* start */)] = edges[i].GetWeight();
+      // Saving weight of current edges for returning in the next iterations.
       m_savedWeight[edges[i].GetTarget()] = edges[i].GetWeight();
-      // Для родетельского joint'a мы знаем его вес без последнего сегмента, так
-      // как для всех детей он будет разным, но(!) мы этот вес сохранили при вычислении ребер
-      // и для каждого ребенка вес родительсого сегмента лежит в |parentWeights[i]|
+      // For parent JointSegment we know it weight without last segment, because for each child
+      // it will differ (child_1 => parent != child_2 => parent), but (!) we save these weight in
+      // |parentWeights[]|. So the weight of i-th edge it is cached "weight of parent JointSegment" +
+      // "parentWeight[i]".
       CHECK_LESS(i, parentWeights.size(), ());
       edges[i].GetWeight() = weight + parentWeights[i];
     }
 
-    // Вес родительского joint'a нам больше не нужен, поэтому удаляем его из std::map.
+    // Delete useless weight of parent JointSegment.
     m_savedWeight.erase(it);
   }
   else
   {
-    // Этот цикл для прямого поиска нужен для правильного веса фейковых joint'ов.
+    // This needs for correct calculating weights of FakeJointSegments in forward A* search.
```

Comment:
s/calculation/calculation
weight calculation of

Reply:
done